Parsers/ASimUserManagement/Parsers/ASimUserManagement.yaml

```diff
@@ -39,6 +39,7 @@ ParserQuery: |
       union isfuzzy=true
           vimUserManagementEmpty,
           ASimUserManagementMicrosoftSecurityEvent (ASimBuiltInDisabled or ('ExcludeASimUserManagementMicrosoftSecurityEvent' in (DisabledParsers))),
+          ASimUserManagementMicrosoftWindowsEvent (ASimBuiltInDisabled or ('ExcludeASimUserManagementMicrosoftWindowsEvent' in (DisabledParsers))),
```

Comment:
please increase union parser version to 0.1.2

Reply:
done